libraries/chain/webassembly/eos-vm-oc/executor.cpp

```diff
@@ -171,14 +194,16 @@ void executor::execute(const code_descriptor& code, const memory& mem, apply_con
    cb->ctx = &context;
    executors_exception_ptr = nullptr;
    cb->eptr = &executors_exception_ptr;
-   cb->current_call_depth_remaining = eosio::chain::wasm_constraints::maximum_call_depth+2;
+   cb->current_call_depth_remaining = max_call_depth + 1;
    cb->current_linear_memory_pages = code.starting_memory_pages;
+   cb->max_linear_memory_pages = max_pages;
    cb->first_invalid_memory_address = code.starting_memory_pages*64*1024;
    cb->full_linear_memory_start = (char*)mem.full_page_memory_base();
    cb->jmp = &executors_sigjmp_buf;
    cb->bounce_buffers = &executors_bounce_buffers;
    cb->running_code_base = (uintptr_t)(code_mapping + code.code_begin);
    cb->is_running = true;
+   cb->globals = globals;
```

Comment:
I'm having trouble finding where this gets used.

Reply:
If I remember correctly, it's accessed from LLVMEmitIR.cpp when setting up the pointers for globals.